Dockerfile

```diff
@@ -1,15 +1,37 @@
-FROM golang:alpine
+ARG GO_VERSION=1.17
+ARG ALPINE_VERSION=3.15
```

Comment:
If we're doing this for reproducibility, we should specify the full image versions, e.g. go 1.17.6 and alpine 3.15.0

Reply:
In general I don't think going as far as locking the exact minor version is needed. It'd be additional work for important security fixes. Minor version isn't specified either in [go.mod](https://github.com/accrescent/devportal/blob/main/go.mod#L3).

I mean I can change this, but are you sure?